### PR TITLE
Add more beta disclaimers for App catalogs

### PR DESCRIPTION
**Pull request by LevFlavien**
Since (beta) is displayed on Apps doc, I thought it would be consistent to display it everywhere else

### CODE REVIEW

File: tasks/reference.js [hunk 1]
```diff
@@ -76,7 +76,7 @@ function determineCategory(tag){
             return 'Reference entities';
         case 'App catalog':
         case 'App catalog product':
-            return 'App catalogs';
+            return 'App catalogs [beta]';
```
Comment (LevFlavien) [inline tasks/reference.js:79]: using parenthesis makes bootstrap affix plugin bug (the menu won't stick on screen)

Comment (meganelepalud-akeneo) [reply tasks/reference.js:79]: so maybe we should use [beta] instead of (beta) in other menus as well?

Comment (LevFlavien) [reply tasks/reference.js:79]: as you wish

Comment (LevFlavien) [reply tasks/reference.js:79]: done

**Review by meganelepalud-akeneo — APPROVED** [review]

**Comment by meganelepalud-akeneo** [comment]
I directly merged the PR to avoid disturbing you with this. :)

**Pull request merged**